dags/vz_cris_import.py

```diff
@@ -0,0 +1,39 @@
+import os
+import docker
+from datetime import datetime
+
+from airflow.decorators import dag, task
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.environ.get("ENVIRONMENT", 'development') # operating environment
+
+@dag(
+    dag_id="vz-cris-import",
+    description="A process which will import the VZ CRIS data into the database on a daily basis using data on the SFTP endpoint",
+    schedule="0 7 * * *",
+    start_date=datetime(2023, 6, 1),
+    catchup=False,
+    tags=["vision-zero", "cris"],
```

Comment:
could you also add a tag `repo:atd-vz-data`?

Reply:
Great call, thank you for asking. It's in there!